css-sizing-3/Overview.bs

```diff
@@ -620,22 +622,67 @@ Box Edges for Sizing: the 'box-sizing' property</h3>
 		</pre>
 	</div>
 
+	<div class="example">
+		This example uses box-sizing to evenly horizontally split
+		two divs with fixed size borders inside a div container,
+		which would otherwise require additional markup.
+
+		sample CSS:
+		<pre><code class="lang-css">
+		div.container {
+			width:38em;
+			border:1em solid black;
+		}
+
+		div.split {
+			box-sizing:border-box;
+			width:50%;
+			border:1em silver ridge;
+			float:left;
+		}
+		</code></pre>
+
+		sample HTML fragment:
+		<pre><code class="lang-markup">
+		&lt;div class="container"&gt;
+		&lt;div class="split"&gt;This div occupies the left half.&lt;/div&gt;
+		&lt;div class="split"&gt;This div occupies the right half.&lt;/div&gt;
+		&lt;/div&gt;
+		</code></pre>
+
+		demonstration of sample CSS and HTML:
+		<div style="max-width:38em; border:1em solid black"><div style="box-sizing:border-box; width:50%; border:1em silver ridge; float:left">This div should occupy the left half.</div><div style="box-sizing:border-box; width:50%; border:1em silver ridge; float:left">This div should occupy the right half.</div>The two divs above should appear side by side, each (including borders) 50% of the content width of their container. If instead they are stacked one on top of the other then your browser does not support 'box-sizing'.</div>
+	</div>
+
 	Note: Certain HTML elements,
 	such as <{button}>,
 	default to ''border-box'' behavior.
 	See HTML for details on which elements have this behavior.
 
-	In the CSS specifications,
+	In legacy CSS specifications,
 	the terms <a>width</a>, <a>height</a>,
 	<a lt="min width">minimum (min) width</a>,
 	<a lt="min height">minimum (min) height</a>,
 	<a lt="max width">maximum (max) width</a>, and
 	<a lt="max height">maximum (max) height</a>
-	refer to the <a lt="inner size">inner</a> size
+	generally refer to the <a lt="inner size">inner</a> size
 	(<a lt="content box">content-box</a> size)
 	of a <a>box</a>
 	unless otherwise indicated.
 
+	Note: [[CSS-UI-3#box-sizing]] provides an explicit disambiguation of these terms
+	for the <a href="https://www.w3.org/TR/CSS21/visudet.html">Visual formatting model details</a> section of [[CSS2]].
+
+	<div class=advisement>To avoid ambiguities,
+	specification authors should avoid ambiguous terms such as width or height without further qualification,
```

Comment:
“should avoid ambiguous uses of terms such as width or height without further qualification”

because sometimes the disambiguation is formed by context and not needed in a specific instance of the term--it's ambiguous usage that's a problem